pkg/installer/installer.go

```diff
@@ -111,8 +111,8 @@ func Install(ctx context.Context, args *Args) (err error) {
 	}
 
 	if args.KubeVersion.Major() != 1 ||
-		args.KubeVersion.Minor() < 18 ||
-		args.KubeVersion.Minor() > 27 {
+		args.KubeVersion.Minor() < 20 ||
+		args.KubeVersion.Minor() > 30 {
```

Comment:
As current code works for `v1.18`, you could leave minor version as is. We will remove the support when PSP is removed completely.